sdk/src/main/java/io/dapr/client/DaprClientHttp.java

```diff
@@ -192,17 +196,21 @@
       if (method == null || method.trim().isEmpty()) {
         throw new IllegalArgumentException("Method name cannot be null or empty.");
       }
-      byte[] serializedRequestBody = objectSerializer.serialize(request);
 
-      String[] pathSegments = new String[]{ DaprHttp.API_VERSION, "invoke", appId, "method", method };
 
+      String[] methodSegments = method.split("/");
+
+      List<String> pathSegments = new ArrayList<>(Arrays.asList(DaprHttp.API_VERSION, "invoke", appId, "method"));
+      pathSegments.addAll(Arrays.asList(methodSegments));
```

Comment:
@artursouza This change is because of the change https://github.com/dapr/dapr/issues/4008

The PubsubIT was failing because the method name was `messages/ttlTopic` which was encoded as `messages%2FttlTopic` which worked before the previous change was made. 

Let me know if we can do it in a different way.